### PR TITLE
Added seeding script to Continuous Deployment (please check it). 

### PR DESCRIPTION
**Pull request by InfiniteStyles**

### CODE REVIEW

File: client/src/components/merchant/MerchantProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const CustomerProfile = props => (
```
Comment (bkleung89) [inline client/src/components/merchant/MerchantProfile.jsx]: Should this say MerchantProfile?

File: client/src/containers/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { Redirect } from 'react-router';
+import { Auth } from '../Auth/Auth.js';
+
+const auth = new Auth();
+
+import { connect } from 'react-redux';
+import { fetchProducts } from '../actions/productActions.jsx';
+import { changeVideo, changeBroadcastMessage, selectFeaturedProduct } from '../actions/broadcastActions.jsx';
+
+class BroadcastView extends React.Component {
```
Comment (bkleung89) [inline client/src/containers/Profile.jsx]: Is the class name appropriate?

File: .circleci/config.yml [hunk 3]
```diff
@@ -30,6 +30,8 @@ jobs:
 
       - run: git checkout package-lock.json
 
+      - run: node seeding/script.js
```
Comment (bkleung89) [inline .circleci/config.yml]: Sorry, I've never tried integrating seeding with CI. If you don't want to spend time troubleshooting this we can always seed the deployed version manually.

File: client/src/containers/Profile.jsx (hunk 2 above)
Comment (InfiniteStyles) [reply client/src/containers/Profile.jsx]: No, Sorry. They were just boilerplates and I forgot to convert the names. Should be fixed now.

**Pull request closed without merging**